the-solutions-journal.csl

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>The Solutions Journal</title>
+    <id>http://www.zotero.org/styles/the-solutions-journal</id>
+    <link rel="self" href="http://www.zotero.org/styles/the-solutions-journal"/>
+    <link rel="template" href="http://www.zotero.org/styles/nature"/>
+    <link href="http://www.thesolutionsjournal.com/node/457" rel="documentation"/>
+    <author>
+      <name>Jafet Andersson</name>
+      <uri>http://www.smhi.se/en/research/research-departments/hydrology/jafet-andersson-1.18716</uri>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="science"/>
+    <category field="generic-base"/>
+    <eissn>2154-0926</eissn>
+    <issnl>2154-0896</issnl>
+    <updated>2015-12-08T11:16:17+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="title">
+    <choose>
+      <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+        <text variable="title" text-case="capitalize-all" strip-periods="false" font-style="italic" font-variant="normal"/>
+      </if>
+      <else-if type="chapter" match="any"/>
+      <else>
+        <text variable="title" suffix="."/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="author">
+    <names variable="author" font-variant="normal">
+      <name delimiter="," and="symbol" et-al-min="5" initialize-with="" name-as-sort-order="all"/>
```

Comment:
please remove  the `et-al-min="5"` here. We commonly place these on the bibliography or citation, and you actually already have `et-al-min="6"`, which is correct (the instructions are "more than 5" i.e. min 6)